include/lsst/afw/detection/Bootprint.h

```diff
@@ -74,6 +74,10 @@ class Bootprint : public lsst::daf::base::Citizen,
     explicit Bootprint(std::shared_ptr<geom::SpanSet> inputSpans,
                        afw::table::Schema const & peakSchema,
                        geom::Box2I const & region=geom::Box2I());
+    explicit Bootprint(): lsst::daf::base::Citizen(typeid(this)),
+                          _spans(std::make_shared<geom::SpanSet>()),
+                          _peaks(PeakTable::makeMinimalSchema()),
+                          _region(geom::Box2I()) {}
```

Comment:
I remember doing in the body when I came across some error, but since this works I can not figure out what could have stopped the other from working. I will make them consistent.